src/main/java/http_server/Router.java

```diff
@@ -137,32 +129,45 @@ public boolean methodInvalid(String route, String method) {
         return routes.get(route) == null ? new HashMap<>() : routes.get(route);
     }
 
-    public void fillResponseForRequest(Request request, Response response) {
+    public void fillResponse(Request request, Response response) {
         getMethodCollection(request.getRoute()).get(request.getMethod()).run(request, response);
     }
 
     private String trimPath(String path) {
-        int trimFrom = path.lastIndexOf(".");
-        return path.substring(trimFrom + 1);
+        if (path.startsWith(".") || path.startsWith("/")) {
+            int trimFrom = path.lastIndexOf(".");
+            return path.substring(trimFrom + 1);
+        } else {
+            return path;
+        }
+
     }
 
     public void staticDirectory(String directoryPath) {
```

Comment:
I noticed that you renamed a method that creates a directory to eliminate the term `static` - should this be renamed as well?